src/rhsmlib/facts/virt.py

```diff
@@ -34,7 +32,7 @@ def get_virt_info(self):
         virt_dict = {}
 
         try:
-            host_type = compat_check_output('/usr/sbin/virt-what')
+            host_type = subprocess.check_output('/usr/sbin/virt-what')
```

Comment:
this returns `bytes` now, so the `splitlines()` below will not work

Reply:
I should have read properly and completely what the `compat_check_output` did. Thanks.